src/server/game/Maps/TransportMgr.cpp

```diff
@@ -423,13 +423,13 @@ void TransportMgr::SpawnContinentTransports()
     if (_transportTemplates.empty())
         return;
 
-    uint32 oldMSTime, count = 0;
-    oldMSTime = getMSTime();
-    QueryResult result;
+    uint32 count = 0;
+    uint32 oldMSTime = getMSTime();
+    QueryResult result = WorldDatabase.Query("SELECT guid, entry FROM transports");
```

Comment:
Why is `result` exposed if there's `CONFIG_ENABLE_CONTINENT_TRANSPORT`, on other words, why did you move it outside if there's no use for it code below in case `CONFIG_ENABLE_CONTINENT_TRANSPORT` false (is disabled) ?

Reply:
Result was always defined with the wdb query to begin with even before this pr. Please test and look over it in debugger yourself before and after the pr adjustments. Both configs were tested and are still functional with no further issues.